src/set_theory/game/nim.lean

```diff
@@ -141,6 +141,16 @@ lemma move_right_nim {o : ordinal} (i) :
   (nim o).move_right (to_right_moves_nim i) = nim i :=
 by simp
 
+/-- A recursion principle for left moves of a nim game. -/
+@[elab_as_eliminator] def nim_left_moves_rec_on {o : ordinal} {P : (nim o).left_moves → Sort*}
+  (i : (nim o).left_moves) (H : ∀ a < o, P $ to_left_moves_nim ⟨a, H⟩) : P i :=
```

Comment:
Have you tested if this works with `induction ... using`? It looks to me like the argument might need reordering?

Reply:
Probably there is no fix

Reply:
So what's to be done here?